README.md

```diff
@@ -1,15 +1,65 @@
 [![Travis](https://api.travis-ci.org/bids-standard/bids-examples.svg?branch=master "Travis")](https://travis-ci.org/bids-standard/bids-examples)
 
-# BIDS-examples
-A set of BIDS compatible datasets with **empty raw data files** that can be used for writing lightweight software tests.
+# bids-examples
+
+This repository contains a set of BIDS compatible datasets with **empty raw
+data files**. These datasets can be useful to:
+
+1. write lightweight software tests
+1. serve as an example on how a BIDS dataset can be structured
 
 ALL RAW DATA FILES IN THIS REPOSITORY ARE EMPTY!
 
-- You can find full versions of most MRI datasets in this repository [here](https://drive.google.com/drive/u/0/folders/0B2JWN60ZLkgkMGlUY3B4MXZIZW8).
-- You can find the full version of mne_sample_data [here](https://drive.google.com/drive/folders/0B_sb8NJ9KsLUQ3BMS0dxZW5nSHM).
-- You can find the full version of ds000246 (Brainstorm auditory tutorial) [here](https://openneuro.org/datasets/ds000246/versions/00001).
-- You can find the full version of ieeg_epilepsy (Brainstorm epileptogenicity tutorial) [here](https://openneuro.org/datasets/ds001779).
+To learn more about the Brain Imaging Data Structure (BIDS), visit
+http://bids.neuroimaging.io/
+
+If you want to contribute a dataset to the examples, we would be happy to
+accommodate it, as long as it exhibits some structure or particularity that is
+not covered by the already present examples so far.
+
+# Dataset index
 
-To learn more about the Brain Imaging Data Structure (BIDS), visit http://bids.neuroimaging.io/
+If you want to update this table, please open a new issue. For simple editing,
+this table is maintained in a [Google sheet](https://docs.google.com/spreadsheets/d/1d9KfmaKvTOpFWgi0Kd-tZGgVTsZk_4BHH9prcgFeigw/edit?usp=sharing), which is then converted to Markdown for this README file.
```

Comment:
I would not keep the google sheet version available. For this initial formatting it is fine and helps a lot, but the google sheet and the markdown table on github are destined to drift apart, making automatic conversion from one to the other impossible.

Reply:
good point :+1: